### PR TITLE
Added a dark scheme to custom color 'bg' and removed fixed color uses

### LINKED ISSUE #26: Make application compatible with dark mode/night mode.
Have all aspects of application compatible with dark mode. Right now lots of text and objects are low visibility if dark mode is toggled.

### PR DESCRIPTION
**Pull request by arota18**
resolve #26

### CODE REVIEW

File: SwiftJym/Assets.xcassets/Bg.colorset/Contents.json [hunk 1]
```diff
@@ -23,9 +23,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0.969",
-          "green" : "0.949",
-          "red" : "0.949"
+          "blue" : "0.039",
+          "green" : "0.039",
+          "red" : "0.039"
```
Comment (arota18) [inline SwiftJym/Assets.xcassets/Bg.colorset/Contents.json:28]: I've picked a slightly lighter black (someting like what u did with the light scheme) but can be changed obviously

File: SwiftJym/Views/ExercisesViews/ExercisesView.swift [hunk 2]
```diff
@@ -83,7 +83,7 @@ struct SearchAndScanView: View {
                 TextField("Search exercises", text: $search)
             }
             .padding(.all, 15)
-            .background(Color.white)
+            .background(Color("bg"))
```
Comment (arota18) [inline SwiftJym/Views/ExercisesViews/ExercisesView.swift:86]: the point of the dark scheme for 'bg' was to enable its use without having to worry about the colorScheme

File: SwiftJym/Assets.xcassets/Bg.colorset/Contents.json (hunk 1 above)
Comment (soaapp) [reply SwiftJym/Assets.xcassets/Bg.colorset/Contents.json:28]: Good morning to you my friend. The colour works great, thank you.

**Review by soaapp — APPROVED** [review]
Approved dark scheme changes. Thank you again.

**Pull request merged**